app/schema.graphql

```diff
@@ -492,7 +493,7 @@ type SidebarGroup {
 input SidebarGroupInput {
   name: String!
   paths: [String!]
-  expanded: Boolean = true
+  expanded: Boolean = null
```

Comment:
Why was this changed?

Reply:
I didn't change this. The schema is autogenerated from the code...

Reply:
The default should be null. It is correct. It was never regenerated during sidebar updates